railties/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   The scaffold generator now generates valid html when the model has_many_attached.
```

Comment:
Changelog is not necessary for this change, and can you please [squash your commits](https://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#squashing-commits)?

Reply:
Thanks for reviewing. done!